src/Components/Server/src/ComponentsServerOptions.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Components.Server
+{
+    /// <summary>
+    /// Options to configure ASP.NET Core Components.
+    /// </summary>
+    public class ComponentsServerOptions
```

Comment:
If we already have some public types named Circuit, CircuitHandler, etc. I would just name this CircuitOptions. It describes it more accurately and is inline with the terms used in other places.